### DISCUSSION TITLE
v0.1

### DISCUSSION THREAD

**Discussion by pajowu** [bugbakery/audapolis#158] [Announcements]
This is our first proper release. While there are still some known bugs and many rough edges, we think it's good enough for some use-cases and serves as a good impression of what audapolis can become.

## What's Changed Since The Last Release
* 🐛 Fix curosr bug when clicking while playing by @pajowu in https://github.com/audapolis/audapolis/pull/133
* 📖️ correct authors in pyproject.toml by @anuejn in https://github.com/audapolis/audapolis/pull/141
* 💚 Specify macos version by @pajowu in https://github.com/audapolis/audapolis/pull/150
* 🔗 add links to survey and funding in-app by @anuejn in https://github.com/audapolis/audapolis/pull/149
* ⚡️ Speedup cursor position calculation by @pajowu in https://github.com/audapolis/audapolis/pull/145

**Full Changelog**: https://github.com/audapolis/audapolis/compare/v21.11.29-1539...v21.12.13-2230

<hr /><em>This discussion was created from the release <a href='https://github.com/audapolis/audapolis/releases/tag/v21.12.13-2230'>v0.1</a>.</em>

**Comment by aplatypus** [comment]
* No fan of appimage myself.  I guess it matters little when you don't have shared libraries, etc. (Just a personal preference).
